api/server/router/build/build_routes.go

```diff
@@ -92,6 +50,7 @@ func newImageBuildOptions(ctx context.Context, r *http.Request) (*types.ImageBui
 	options.CPUSetCPUs = r.FormValue("cpusetcpus")
 	options.CPUSetMems = r.FormValue("cpusetmems")
 	options.CgroupParent = r.FormValue("cgroupparent")
+	options.Tags = r.Form["t"]
```

Comment:
Sounds good @darrenstahlmsft . So until Windows CI is fully automated, can we ping you to run regression tests for the "builder" subsystem? Thanks.